mava/systems/jax/ippo/config.py

```diff
@@ -16,10 +16,19 @@
 """Default hyperparameters for IPPO system."""
 from dataclasses import dataclass
 
+from mava.components.jax import executing
+
 
 @dataclass
 class IPPODefaultConfig:
     sample_batch_size: int = 512
     sequence_length: int = 20
     period: int = 10
     use_next_extras: bool = False
+
+
+# The components that a needed for IPPO with a recurrent policy.
+recurrent_policy_components = [
```

Comment:
Also, these components are not in IPPO folders so perhaps these groupings should not be in an IPPO class